flexget/components/notify/notifiers/cronitor.py

```diff
@@ -0,0 +1,91 @@
+from builtins import *
```

Comment:
Missing future? Absolute import etc.

Reply:
not sure what you mean

Reply:
`from __future__ import unicode_literals...` I think he means.